src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/Hover/RemoteHoverService.cs

```diff
@@ -111,7 +111,8 @@ protected override IRemoteHoverService CreateService(in ServiceArgs args)
                 }
             }
 
-            return Results(csharpHover);
+            // As there is a C# hover, stop further handling.
+            return new RemoteResponse<Hover?>(StopHandling: true, Result: csharpHover);
```

Comment:
I might be missing it, but I was expecting StopHandling to be used when we product a Razor response at times too. eg, when hovering over `<PageTitle>` in a .razor file, we don't need to make a Html request. In fact, I can't think of any scenario where we'd want to call the Html server if we're in a .razor file and we were able to produce a hover response ourselves. It's really only MVC that has Razor concepts apply to regular Html elements or attributes.

Reply:
Ohh of course, because thats actually a C# hover. Okay, cool. You lost the fight over who missed something :P